.gitignore

```diff
@@ -465,8 +465,8 @@ lib/cpluff/stamp-h1
 /tools/android/packaging/xbmc/strings.xml
 
 #/tools/depends
+/tools/depends/native/*/*native/
 /tools/depends/native/JsonSchemaBuilder/bin/
-/tools/depends/native/JsonSchemaBuilder/native/
 /tools/depends/target/ffmpeg/.ffmpeg-installed
 /tools/depends/target/ffmpeg/ffmpeg-2.4-Helix-alpha4.tar.gz
 /tools/depends/target/ffmpeg/ffmpeg-2.4-Helix-alpha4/
```

Comment:
Sure, no worries, done :) Is this generic enough? I tried to make it not to generic to avoid excluding too much though we could expand it even more? Maybe even exclude tar.gz files globally in the repo?